lua/swampshop/gizmos/paint.lua

```diff
@@ -0,0 +1,226 @@
+--[[
```

Comment:
do we... need this?

Reply:
nah it was more something i wanted to try, but i feel like there's better tools for reskinning out there . i commented it out and am probably just gonna delete it